mobile/src/libs/components/input-file/libs/request-media-permissions.helper.ts.ts

```diff
@@ -0,0 +1,35 @@
+import { PermissionsAndroid } from 'react-native';
```

Comment:
should be in `input-file/libs/helpers/request-media-permissions.helper.ts` and re-exported from helpers.ts